### PR TITLE
enable webpack loader for css and fonts

### PR DESCRIPTION
**Pull request by gautamsi**
* adds standard loaders for css (`style-loader` and `css-loader`)
* enable loading of fonts (with existing `file-loader` config)

I needed this for importing css from node_modules, which in tern loads the fonts using `@font-face`

```jsx
import 'path_to_css_file.css'
```

### CODE REVIEW

**Comment by MadeByMike** [comment]
So you would place: `import 'path_to_css_file.css'` in a custom page?

Did you test this? The webpack config is relative to the admin-ui package directory ... however the CSS loader might work differently?

**Comment by gautamsi** [comment]
I have it running in dev now. It will only load `imports` from custom pages. anything outside of it (like app-next or custom react app) will not use loader from `app-admin-ui`

**Comment by mitchellhamilton** [comment]
The resolving of this should work fine since everything will be resolved relative to the custom page.

**Comment by gautamsi** [comment]
> So you would place: `import 'path_to_css_file.css'` in a custom page?
> 
> Did you test this? The webpack config is relative to the admin-ui package directory ... however the CSS loader might work differently?

I have tested two ways to do this, both works well

**Relative path to the file where you are importing**
```jsx
import '../../../public/slick/slick.css';
import '../../../public/slick/slick-theme.css';
```

**path in node_module package**
```jsx
import 'slick-carousel/slick/slick.css';
import 'slick-carousel/slick/slick-theme.css';
```

**Comment by MadeByMike** [comment]
Ok I think I am down with this :)

File: .changeset/ninety-ways-remain.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@keystonejs/app-admin-ui': minor
+---
+
+Allow loading of fonts and add loader for css styles
```
Comment (MadeByMike) [inline .changeset/ninety-ways-remain.md]:
```suggestion
Added a font loader to the webpack configuration that will allow loading of css files including in custom pages
```

**Comment by MadeByMike** [comment]
@gautamsi Just need to merge the changes to the changeset message. 

The reason is, in the interest of making readable changelogs we're trying to use past tense and always write in complete sentences.

**Comment by MadeByMike** [comment]
BTW if you would like us to merge minor things like this for you, feel free to allow commits to add some of us as contributors to your fork or change the permission settings.

We'll never commit anything beyond a simple typo or incidental changes like this without discussing.

If I had this I'd merge this PR now :)

**Comment by gautamsi** [comment]
I never remove this checkbox, @jesstelford and @timleslie were able to merge in my PR earlier, I believe you did too. Not sure where can  look for permission.

image

**Pull request merged**